### ISSUE TITLE
[Question] How to connect to database using serverless/cloud function.

### ISSUE THREAD

**Issue by caiiiycuk** [ydb-platform/ydb-nodejs-sdk#189]
Hi, I read all documentation that I found, but still can't connect to ydb. I use serverless to deploy the stack.

I use following code to connect:
  ```js
        const driver = new Driver({
            endpoint: "grpcs://ydb.serverless.yandexcloud.net:2135",
            database: "/ru-central1/b1g5f2a3d9hru8cnrec5/etnsmorj75fl505b6t4g",
            authService: new MetadataAuthService(),
        });

        const ready = await driver.ready(5);

        return {
            statusCode: 200,
            headers: {
                'Content-Type': 'text/plain',
            },
            body: "ready: " + ready,
        }
```

However when I try to execute the handler, I have following error:
```
No certificate found
It seems that you are using grpcs (secure) endpoint in a bundled environment.
Either provide YDB_SSL_ROOT_CERTIFICATES_FILE environment variable
or copy contents of ydb-nodejs-sdk/certs to ./certs path relative to the bundled file
```

This message looks really weird for me. Because:
1. The npm package is called 'ydb-sdk' not a 'ydb-nodejs-sdk'
2. There is no such folder 'ydb-sdk/certs'
3. I found the 'certs' folder in this repository, but next '**copy contents of ydb-nodejs-sdk/certs to ./certs path relative to the bundled file**'. How I can do it with serverless? I think it's not trivial.
4. Anyway why I need to deal with certs if I build a cloud solution? I have aws background, so if I have a role to access the database then I can access it without auth, that is. 

For accessing database my cloud function have '-editor' role, but no luck. Can you help me?

P. S. The examples I found looks outdated, and not well structured.

**Comment by zeruk** [comment]
Hi! Sorry hear about that, it was a bug, we fixed it right now, in version [ydb-sdk@3.4.4](https://www.npmjs.com/package/ydb-sdk/v/3.4.4). Please update deps in your package and check, if it works.

**Comment by zeruk** [comment]
As I see, you use `MetadataAuthService`, I'm not sure that you need it.
`TokenAuthService` worked fine for me, when I used ydb.serverless.yandexcloud.net. You'll need to use token from `yc iam create-token` like that: const authService = new TokenAuthService('t1.9euelZ....')
